### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by KerstenBreuer**
Documented high level functions.

### CODE REVIEW

File: datameta_client/submissions.py [hunk 1]
```diff
@@ -91,7 +100,15 @@ def submit(
     url:Optional[str] = None, 
     token:Optional[str] = None,
     quiet:bool = False,
-) -> bool:
+) -> dict:
+    """Submit a set of metadatasets and a set of files.
+
+    This requires a list of metadataset IDs and a list of file IDs. The IDs can be either
+    given as a comman-separated string or as a Python list of string (the latter is not
```
Comment (cschu) [inline datameta_client/submissions.py]:
```suggestion
    given as a comma-separated string or as a Python list of string (the latter is not
```

**Comment by KerstenBreuer** [comment]
Thanks @cschu :)

**Comment by cschu** [comment]
> Thanks @cschu :)

I think I found my niche. I'm here to nitpick on typos XD

**Comment by KerstenBreuer** [comment]
@lkuchenb can be merged?

**Pull request merged**